test/e2e/framework/pod/delete.go

```diff
@@ -44,7 +44,7 @@ func DeletePodOrFail(ctx context.Context, c clientset.Interface, ns, name string
 		return
 	}
 
-	expectNoError(err, "failed to delete pod %s in namespace %s", name, ns)
+	framework.ExpectNoError(err, "failed to delete pod %s in namespace %s", name, ns)
```

Comment:
```suggestion
	framework.ExpectNoErrorWithOffset(1, err, "failed to delete pod %s in namespace %s", name, ns)
```

We don't want to log the helper package as the location where things went wrong.

Same below.